mlir/test/Target/Wasm/memory_min_eq_max.mlir

```diff
@@ -0,0 +1,7 @@
+// RUN: yaml2obj %S/inputs/memory_min_eq_max.yaml.wasm -o - | mlir-translate --import-wasm | FileCheck %s
+
+/* Source code used to create this test:
+(module (memory 0 0))
+*/
+
+// CHECK-LABEL:   "wasmssa.memory"() <{limits = !wasmssa<limit[0: 0]>, sym_name = "mem_0", sym_visibility = "nested"}> : () -> ()
```

Comment:
I just notice that many ops don't have a `let assemblyFormat =` defined, is this intended?

Reply:
It is indeed an oversight, thanks for pointing it.

Reply:
I'll let you fix this in another PR then! Closing here.

Reply:
Done in PR #152668